guides/v2.1/cloud/project/new-relic.md

```diff
@@ -18,13 +17,46 @@ New Relic APM provides the following features to Magento:
+By default, the Pro Staging and Production environments include New Relic. If it is not installed, submit a [support ticket]({{ page.baseurl }}/cloud/trouble/trouble.html) to request installation. {{site.data.var.ece}} Pro plan customers also need to submit a support ticket to use New Relic on an Integration branch. 
+
+
+{:.bs-callout .bs-callout-info}
+Your Magento Commerce Cloud account includes the software for the New Relic APM agent. You do not need to purchase or install the New Relic extension, which is different than the New Relic APM service. 
+
+## New Relic account credentials
+
+When Magento provisions your {{site.data.var.ece}} project, the Project Owner should receive an email from New Relic with the credentials and instructions for accessing the account. If you did not receive the email, you can use the Project Owner email address to [reset the password](https://rpm.newrelic.com/forgot_password){:target="_blank"}.
+
+If you cannot access the account, submit a [Magento support ticket]({{ page.baseurl }}/cloud/trouble/trouble.html).
+
+
+## Add account users
+
+{{site.data.var.ece}} Project Owners and Admin users can add and remove users from the New Relic account.
+
+We recommend adding at least one account through New Relic to manage all access, integrations, and usage of the tool. We also recommend promoting one of the added members to Admin, to manage all New Relic access and integrations.
+
+1.  Using your Project Owner New Relic credentials, log in to [New Relic](https://login.newrelic.com/login){:target="_blank"}.
+
+1.  From the account dropdown, select **Account settings > Account > Users and roles**.
+
+1.  Click **New user**.
+
+1.  Add the name and email address for the account.
+
+1.  Assign the user role: *Admin*, *User*, or *Restricted*.
+
+1. Click **Add user**.
+
+1. Ask the new user to check their email for a New Relic notification with account information.
+
+
+See [Accounts](https://docs.newrelic.com/docs/accounts/accounts) in the New Relic documentation for information about user roles, managing user accounts, and changing account ownership.
 
-By default, the Pro Staging and Production environments include New Relic. If it is not installed, submit a [support ticket]({{ page.baseurl }}/cloud/trouble/trouble.html) to request installation.
 
 ## Add New Relic extension to your project
 
-You must list the New relic extension in the `.magento.app.yaml` file:
+If the `.magento.app.yaml` file for your project does not include the `newrelic` extension, you must add it. 
```

Comment:
The previous version was more clear and straightforward.

Reply:
I agree that it is less straightforward, but I changed it because it is possible that the newrelic extesnion is already there. They are adding it to the magento-cloud template, so some customers might have it already, and some might not have it.  Basically, they need to verify that it's there, and add it if it is missing.